testing/sdk_plan.py

```diff
@@ -155,8 +157,10 @@ def fn():
         failures = sdk_tasks.get_failed_task_count(service_name)
         if failures - initial_failures > MAX_NEW_TASK_FAILURES:
             log.error(
-                "Service %s exceeded failures increase while doing %s and trying to reach %s",
+                "Tasks in service %s failed %d times since starting %ds ago to wait for %s to reach %s, aborting.",
```

Comment:
you may want to append the letter `s` to `wait for %s to reach` else your message will look something like
`wait for 26.252 for deploy to reach COMPLETE`

Reply:
ah you're right, sorry about that